app/javascript/components/pages/Meetups.jsx

```diff
@@ -36,45 +36,59 @@ MonthSection.propTypes = {
-                {speakers.length > 0
-                    ? speakers.map(({ id, name, tagline, image_url }) => (
-                          <div key={id} className="flex content-center mb-8 text-lg">
-                              <img
-                                  className="object-cover w-14 h-14 mr-4 rounded-full"
-                                  src={image_url}
-                                  alt=""
-                              />
-                              <div>
-                                  <p className="font-bold text-gray md:text-lg">{name}</p>
-                                  <p className="text-sm text-gray md:text-lg">{tagline}</p>
-                              </div>
-                          </div>
-                      ))
-                    : null}
-            </div>
-            <div className="bg-gray-200 text-right">
-                <button className="my-4 mr-6 py-4 px-8 bg-gray-600 rounded text-white text-lg md:text-xl">
-                    View
-                </button>
+const Meetup = ({ speakers, title = '', event_speakers }) => {
+    const eventWithSpeaker = event_speakers.map((talk) => {
+        const speaker = speakers.find((speak) => speak.id === talk.speaker_id);
+        return { ...talk, speaker };
+    });
+    return (
+        <li>
+            <div className="meetups__card flex flex-col pb-12 w-[600px]">
+                <div className="w-full rounded shadow-lg border-t p-10 border-gray-100 overflow-hidden">
+                    <h4 className="mb-4 text-xl font-bold text-gray md:text-2xl">{title}</h4>
+                    {eventWithSpeaker.length > 0 &&
+                        eventWithSpeaker.map(({ id, talk_title, speaker }) => (
+                            <div key={id}>
+                                <h5 className="mb-4 text-xl font-bold text-gray md:text-xl">
+                                    {talk_title}
+                                </h5>
+                                <div className="flex content-center mb-8 text-lg">
+                                    <img
+                                        className="object-cover w-14 h-14 mr-4 rounded-full"
+                                        src={speaker.image_url}
+                                        alt=""
+                                    />
+                                    <div>
+                                        <p className="font-bold text-gray md:text-lg">
+                                            {speaker.name}
+                                        </p>
+                                        <p className="text-sm text-gray md:text-lg">
+                                            {speaker.tagline}
+                                        </p>
+                                    </div>
+                                </div>
+                            </div>
+                        ))}
+                </div>
+                <div className="bg-gray-200 text-right">
```

Comment:
This is 100% personal preference, and you can totally shut me down, but I'd like to see the View button in the middle, rather than on the right. WDYT?

Reply:
@lhmzhou Thanks for the review!  About the View button, I haven't touched where it was originally which is as per the design, so I guess I would say that if we want to change it that should be a separate issue?

Reply:
Personally I quite like the balance of it on the right as the rest of the page is quite left heavy, but happy to change if you like.

Reply:
That makes sense! We can also poll the team before we make a final decision. 😄